packages/now-cli/test/integration.js

```diff
@@ -2465,15 +2535,3 @@ test('whoami with local .now scope', async t => {
   // clean up
   await remove(path.join(directory, '.now'));
 });
-
-test.after.always(async () => {
-  // Make sure the token gets revoked
-  await execa(binaryPath, ['logout', ...defaultArgs]);
-
-  if (!tmpDir) {
-    return;
-  }
-
-  // Remove config directory entirely
-  tmpDir.removeCallback();
-});
```

Comment:
Is this intentional?

Reply:
Yes, I just moved it up

Reply:
https://github.com/zeit/now/pull/3686/files/86bbcd36d27ccb010c22a2b796e2a29297f25a3f#diff-6324a33fb2da36e19435c2c6c5368767R216